app/Models/Question.php

```diff
@@ -157,7 +197,7 @@ public function mentions(): Collection
             return $mentionedUsers;
         }
 
-        preg_match_all("/@([^\s,.?!\/@<]+)/i", type($this->content)->asString(), $contentMatches);
+        preg_match_all("/@([^\s,.?!\/@<]+)/i", $this->content, $contentMatches);
```

Comment:
I removed this because the type is inferred for static analysis by the docblocks for the content attribute.